code/modules/research/designs/misc_designs.dm

```diff
@@ -603,6 +603,16 @@
 	category = list("Equipment")
 	departmental_flags = DEPARTMENTAL_FLAG_SECURITY
 
+/datum/design/inspector
+	name = "N-Spect Scanner"
+	desc = "Central Command-issued inspection device. Performs inspections according to Nanotrasen protocols when activated, then prints an encrypted report regarding the maintenance of the station."
+	id = "inspector"
+	build_type = PROTOLATHE | AWAY_LATHE
+	materials = list(/datum/material/iron = 5000, /datum/material/glass = 1000)
```

Comment:
add a more advanced material to this recipe, readding the "hard to replace" up to you. Without any precious metal usage this jumps from a nonreplenishable device to a near infinite one.

Reply:
Do you have a suggestion for what? Also, is there a downside to an item only useful for scanning areas for bounties being near infinite?

Reply:
Added a cost in gold and uranium, plus a line about how the machine is definitely not giving you cancer.